cdci_data_analysis/flask_app/app.py

```diff
@@ -221,6 +227,26 @@ def common_exception_payload():
     return payload
 
 
+
+class ExitStatus(Schema):
+    status = fields.Int(validate=OneOf([0, 1]))
+    message = fields.Str(desciption="if query_status == 'failed', shown in waitingDialog in red")     
```

Comment:
I guess it's description

Reply:
you guess correctly! It would fail if it was applied.